pastel/reg_ticket.go

```diff
@@ -54,18 +71,155 @@ type AppTicket struct {
 	ArtworkKeywordSet              string `json:"artwork_keyword_set"`
 	TotalCopies                    int    `json:"total_copies"`
 
-	ThumbnailHash []byte `json:"thumbnail_hash"`
-	DataHash      []byte `json:"data_hash"`
+	PreviewHash    []byte `json:"preview_hash"`
+	Thumbnail1Hash []byte `json:"thumbnail1_hash"`
+	Thumbnail2Hash []byte `json:"thumbnail2_hash"`
+
+	DataHash []byte `json:"data_hash"`
+
+	Fingerprints          []float64 `json:"fingerprints"`
```

Comment:
as comment from ayk then `Fingerprints` should be removed ?

Reply:
no, fingerprints now is an array of floats